integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -273,16 +362,17 @@ def export(self, **kwargs):
                 for item in content_metadata_items:
                     key = get_content_metadata_item_id(item)
 
-                    # Used to save modified times and catalog uuid's to sent transmission records
```

Comment:
this mapping just tracked the catalog modified at datetime as well as the catalog uuid. since we are precreating `ContentMetadataItemTransmission` items, we can store this information on those records and drop this extra dict.